node/src/actors/chain_manager/mod.rs

```diff
@@ -1331,13 +1334,11 @@ impl ChainManager {
                         let reputation_engine = act.chain_state.reputation_engine.as_ref().unwrap();
                         let last_superblock_signed_by_bootstrap = last_superblock_signed_by_bootstrap(&chain_info.consensus_constants);
 
-                        let is_superblock_hack_period = superblock_index > 750 && superblock_index < 1344;
-
                         let ars_members =
                             // Before reaching the epoch activity_period + collateral_age the bootstrap committee signs the superblock
                             // collateral_age is measured in blocks instead of epochs, but this only means that the period in which
                             // the bootstrap committee signs is at least epoch activity_period + collateral_age
-                            if is_superblock_hack_period {
+                            if is_first_superblock_hack_period(superblock_index) {
                                 // Bootstrap committee
                                 let hack_bootstrapping_committee = [
                                     "wit1asdpcspwysf0hg5kgwvgsp2h6g65y5kg9gj5dz",
```

Comment:
Can we move this list of addresses to the new post_mainnet_validations.rs file

Reply:
done